TestTools/StreamChatTestTools/Fixtures/JSONs/ThreadList.json

```diff
@@ -0,0 +1,5072 @@
+        "updated_at": "2024-03-26T12:02:37.748782Z",
+        "created_by": {
+          "id": "r2-d2",
+          "role": "user",
+          "created_at": "2020-12-07T11:37:35.98087Z",
+          "updated_at": "2024-03-05T21:06:56.824843Z",
+          "last_active": "2024-03-27T12:53:42.462142Z",
+          "banned": false,
+          "online": false,
+          "language": "en",
+          "name": "R2-D2",
+          "image": "https://vignette.wikia.nocookie.net/starwars/images/e/eb/ArtooTFA2-Fathead.png",
+          "birthland": "Naboo",
+          "extraData": {},
+          "image_url": "https://vignette.wikia.nocookie.net/starwars/images/e/eb/ArtooTFA2-Fathead.png"
+        },
+        "frozen": false,
+        "disabled": false,
+        "members": [
+          {
+            "user_id": "han_solo",
+            "user": {
+              "id": "han_solo",
+              "role": "user",
+              "created_at": "2020-12-07T11:37:34.230369Z",
+              "updated_at": "2024-03-15T11:22:08.390161Z",
+              "last_active": "2024-04-01T16:01:44.46442267Z",
+              "banned": false,
+              "online": true,
+              "language": "en",
+              "extraData": {},
+              "image_url": "https://vignette.wikia.nocookie.net/starwars/images/e/e2/TFAHanSolo.png",
+              "name": "Han Solo",
+              "image": "https://vignette.wikia.nocookie.net/starwars/images/e/e2/TFAHanSolo.png",
+              "birthland": "Corellia"
+            },
+            "status": "member",
+            "created_at": "2024-03-26T11:13:35.162494Z",
+            "updated_at": "2024-03-26T11:13:35.162494Z",
+            "banned": false,
+            "shadow_banned": false,
+            "role": "member",
+            "channel_role": "channel_member",
+            "notifications_muted": false
+          },
+          {
+            "user_id": "r2-d2",
+            "user": {
+              "id": "r2-d2",
+              "role": "user",
+              "created_at": "2020-12-07T11:37:35.98087Z",
+              "updated_at": "2024-03-05T21:06:56.824843Z",
+              "last_active": "2024-03-27T12:53:42.462142Z",
+              "banned": false,
+              "online": false,
+              "language": "en",
+              "name": "R2-D2",
+              "image": "https://vignette.wikia.nocookie.net/starwars/images/e/eb/ArtooTFA2-Fathead.png",
+              "birthland": "Naboo",
+              "extraData": {},
+              "image_url": "https://vignette.wikia.nocookie.net/starwars/images/e/eb/ArtooTFA2-Fathead.png"
+            },
+            "status": "member",
+            "created_at": "2024-03-26T11:13:35.162494Z",
+            "updated_at": "2024-03-26T11:13:35.162494Z",
+            "banned": false,
+            "shadow_banned": false,
+            "role": "owner",
+            "channel_role": "channel_member",
+            "notifications_muted": false
+          },
+          {
+            "user_id": "tommaso-0",
```

Comment:
There are so many users here, we don't really need this, we just need couple of these.

Reply:
Done